test/nist_test.c

```diff
@@ -216,6 +237,9 @@ int main(void)
 
     ret = test_keygen(pk, sk);
 
+    print_hex(pk, CRYPTO_PUBLIC_KEY, "pk");
+    // print_hex(sk, CRYPTO_SECRET_KEY, "sk");
+
```

Comment:
```suggestion
```

Reply:
fixed in [3de9bec](https://github.com/kj3moraes/further-testing-xmss/pull/5/commits/3de9becac52a3876f01b400d59179244cdb65dd2)